src/components/v2/MintRepayVai/Mint/index.tsx

```diff
@@ -58,17 +34,10 @@ export const MintUi: React.FC<IMintUiProps> = ({
     [limitWei],
   );
 
-  const handleSubmit = (values: FormValues) => {
-    if (values.amount) {
-      onSubmit(values.amount);
-    }
-  };
-
   return (
-    // TODO: add validation schema
-    <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-      {({ values, setFieldValue, handleBlur, isSubmitting, isValid }) => (
-        <Form>
+    <AmountForm onSubmit={onSubmit}>
+      {({ values, setFieldValue, handleBlur, isSubmitting, isValid, touched }) => (
+        <>
```

Comment:
is this fragment needed?

Reply:
Yeah, the render function only allows one JSX element to be returned.